scripts/provisioning/configcmd.py

```diff
@@ -87,17 +87,50 @@ def process(self, configure_only_openldap = False, configure_only_haproxy = Fals
         self.configure_openldap()
         self.configure_haproxy()
 
-      # create topic for background delete
-      bgdeleteconfig = CORTXS3Config()
-      if bgdeleteconfig.get_messaging_platform() == MESSAGE_BUS:
-        self.logger.info('Create topic started')
-        self.create_topic(bgdeleteconfig.get_msgbus_admin_id,
-                          bgdeleteconfig.get_msgbus_topic(),
-                          self.get_msgbus_partition_count())
-        self.logger.info('Create topic completed')
+      if self.module == "s3bgdeleteproducer" or self.module == None :
+        # disable BG delete services on reboot as
+        # it will be managed by HA
+        self.logger.info('Disable services on reboot started')
+        services_list = ["s3backgroundproducer", "s3backgroundconsumer"]
+        self.disable_services(services_list)
+        self.logger.info('Disable services on reboot completed')
+
+        #update endpoint and port in s3BG config file
+        self.update_endpoint_and_port()
+
+        #copy config file to /etc/cortx
+        self.copy_config_file()
+
+        # create topic for background delete
+        bgdeleteconfig = CORTXS3Config()
+        if bgdeleteconfig.get_messaging_platform() == MESSAGE_BUS:
+          self.logger.info('Create topic started')
+          self.create_topic(bgdeleteconfig.get_msgbus_admin_id,
+                            bgdeleteconfig.get_msgbus_topic(),
+                            self.get_msgbus_partition_count())
+          self.logger.info('Create topic completed')
     except Exception as e:
       raise S3PROVError(f'process() failed with exception: {e}')
 
+  def update_endpoint_and_port(self):
```

Comment:
Update following key too in to config file. 

      scheduler_schedule_interval: 1200
      max_keys: 1000

Reply:
done